driver-adapters-wasm/pg-cfpages-basic/run.sh

```diff
@@ -7,5 +7,9 @@ pnpm install
 
 pnpm prisma generate
 
```

Comment:
TODO here: Delete the file or folder that we later create via `wrangler pages functions build` so we can run the same command twice here, once for default output and once for custom output.